src/rules/noImplicitDependenciesRule.ts

```diff
@@ -89,6 +91,13 @@ function walk(ctx: Lint.WalkContext<Options>) {
         }
         return dependencies.has(module);
     }
+
+    function isThereSimilarPackage(module: string): boolean {
+        if (dependencies === undefined) {
```

Comment:
when this function is called, `dependencies` will never be undefined. you can simply assert that by using `return dependencies!.has(module.toLowerCase());`